.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+on:
+  push:
+    branches:
+      - main
+      
+name: Release
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: cargo install auto-release
+      - run: auto-release -p ext4-view --condition subject
```

Comment:
Doesn't match the commit message: commit says "when a version bump is merged", but this says "when the commit starts with `release:`".

Maybe add short "Releasing" section to the README?

Reply:
Will add a short doc for it.

It's both: the PR to bump the version should have a subject starting with `release:`.